examples/run_demo.py

```diff
@@ -17,30 +17,41 @@
 #
 # ------------------------------------------------------------------------------
 import argparse
+import json
 import os
 from typing import Optional, Sequence
 
 from colearn.training import initial_result, collective_learning_round, set_equal_weights
 from colearn.utils.plot import ColearnPlot
 from colearn.utils.results import Results, print_results
-from colearn_other.mli_factory import TaskType, mli_factory
+from colearn_grpc.example_mli_factory import ExampleMliFactory
```

Comment:
I refactored this to use the ExampleMliFactory rather than the old mli_factory. This means that the tests that run this script are testing ExampleMliFactory which is more useful than testing mli_factory.

Reply:
👍